docker/README.md

```diff
@@ -0,0 +1,54 @@
+# Instructions for building the Docker container from the Dockerfile and running code within the container
+
+## Starting docker daemon within WSL2 or Linux OS
+`sudo service docker start`
+
+`sudo dockerd > /dev/null 2>&1 &`
+
+
+## Building an image from the Dockerfile
+The path to Dockerfile can be on Windows (/mnt/c) or WSL Linux filesystem. The syntax is:
+
+`docker build -t drb-insal-gs:<tag> <path to Dockerfile>`
+
+The tag should be a version, like `v0.1`. Every new build should use a new version number. Images can have multiple tags. The latest build should also have the `latest` tag.
+
+Windows filesystem:
+
+`docker build -t drb-insal-gs:v0.1 /mnt/c/users/jsmith/OneDrive\ -\ DOI/Documents/Projects/DRB-InlandSalinity/drb-inland-salinity-ml/docker/`
+
+Linux filesystem: I cloned our repo to /home/jsmith/
+
+`docker build -t drb-insal-gs:v0.1 /home/jsmith/drb-inland-salinity-ml/docker/`
+
+
+## Running the container and testing the build of our repo
+- -v mounts your files to the server
+- --rm removes the container after the session ends
+- -e sets your username
+- your password will print to the screen
+- go to localhost:8787 in a browser to access your Rstudio session
+
```

Comment:
I recommend putting these lines after the docker commands (the commands that currently start in line 34). That way people see the commands and then if they are curious to know what the different flags are they can learn more below.

Reply:
I also think wording it like this would be a little clearer (you can edit the wording in step 4 :) )
```suggestion
Explanation of options:
- -v mounts your files to the server
- --rm removes the container after the session ends
- -e sets your username

Once the server has started:
1. your password will print on the screen
2. go to localhost:8787 in a browser
3. put in your username (from the `-e` option above) and password (generated by the server)
4. make some magic with Rstudio in your browser

```